src/python/grpcio/grpc/_cython/_cygrpc/channel.pyx.pxi

```diff
@@ -542,3 +551,13 @@ cdef class Channel:
 
   def close_on_fork(self, code, details):
     _close(self, code, details, True)
+
+  def register_method(self, method, host):
```

Comment:
What does `host` represent here? Is this the remote host, i.e. the target string? If we never pass a value for `host`, will we have issues if we have two concurrent channels pointed at two different target strings but using the same methods? If so, should we be setting a value every time?

If not, and it's safe to unconditionally pass `NULL` here, can we just get rid of the `host` parameter to this method and not have to worry about it?